content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md

```diff
@@ -10,45 +10,160 @@ content_template: templates/concept
 
 {{% capture overview %}}
 
-This document describes how to set up TLS client certificate bootstrapping for
-kubelets. Kubernetes 1.4 introduced an API for requesting certificates from a
-cluster-level Certificate Authority (CA). The original intent of this API is to
-enable provisioning of TLS client certificates for kubelets. The proposal can be
+In a kubernetes cluster, the components on the worker nodes - kubelet and kube-proxy - need to communicate with Kubernetes master components, i.e. kube-apiserver.
```

Comment:
Kubernetes cluster

Reply:
you mean capitalization?

Reply:
As per the style guide, let's try to avoid latin terms such as i.e. :) Maybe you can use 'such as' instead (felt it's more appropriate than 'that is' in this context)?

Reply:
Aw, but I _like_ `id est`! :-)

Sure. Updating to capitalize now and removing `id est`.